core/include/tangram/tile/tileTask.h

```diff
@@ -122,6 +123,8 @@ class BinaryTileTask : public TileTask {
     std::shared_ptr<std::vector<char>> rawTileData;
 
     bool dataFromCache = false;
+
+    UrlRequestHandle urlRequestHandle = -1;
```

Comment:
I think it's better to keep this type opaque and not do logic on its value. We can simply add a boolean here to determine whether the request is active.